app/config/config.yml

```diff
@@ -1,17 +1,21 @@
 imports:
     - { resource: parameters.yml }
+    - { resource: doctrine.yml }
 
 framework:
     secret:          %kernel.secret%
     router:
         resource: "%kernel.root_dir%/config/routing.yml"
         strict_requirements: ~
     form:            ~
+    validation:
+        enable_annotations: true
     csrf_protection: ~
     templating:
         engines: ['twig']
     session:
-      handler_id: ~
+        handler_id: ~
+        storage_id: session.storage.php_bridge
```

Comment:
it was needed to get symfony form running because spoon library boots the session and it gave conflicts